src/System.Net.NetworkInformation/src/System/Net/StartIPOptions.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    [Flags]
+    internal enum StartIPOptions { Both = 3, None = 0, StartIPv4 = 1, StartIPv6 = 2 }
```

Comment:
Sorry, I wasn't clear: this file is needed but it's an older version compared to the one in GitHub.
Please merge with master then manually check that code that got split into separate files has been updated with the changes.